### PR TITLE
SALTO-4713: deploy error reportProgress bugfix

### PR DESCRIPTION
**Pull request by tamtamirr**
Fixed a bug where deploy error reportProgress wouldn't be reported for Errors with empty string.

---
_Release Notes_: 
Core:
- Fixed a bug where deploy error reportProgress wouldn't be reported for Errors with empty string.

---
_User Notifications_: 
_None_

### CODE REVIEW

**Review by ori-moisis — COMMENTED** [review]
This is fixing a bug, there should be a test that verifies it

File: packages/cli/test/commands/deploy.test.ts [hunk 1]
```diff
@@ -253,6 +253,7 @@ describe('deploy command', () => {
         })
         expect(result).toBe(CliExitCode.AppError)
         expect(callbacks.getUserBooleanInput).not.toHaveBeenCalled()
+        expect(output.stderr.content).toContain('Failed')
```
Comment (tamtamirr) [inline packages/cli/test/commands/deploy.test.ts:256]: Rolling back my code, will actually make this expect fail

File: packages/cli/test/mocks.ts [hunk 2]
```diff
@@ -760,7 +760,7 @@ export const deploy = async (
     numOfChangesReported += 1
     if (numOfChangesReported / 3 === 1) {
       reportProgress(change, 'started')
-      reportProgress(change, 'error', 'details')
+      reportProgress(change, 'error', '')
```
Comment (tamtamirr) [inline packages/cli/test/mocks.ts:763]: Changed to empty string to make sure the bug is covered.

**Review by tamtamirr — COMMENTED** [review]
Added some comments for clarification

**Review by benisraeladi — APPROVED** [review]

**Pull request merged**